### DISCUSSION TITLE
GIF transparent background color & HDR

### DISCUSSION THREAD

**Discussion by manipEgo** [rocksdanister/lively#2545] [Q&A] [answered]
Is there a way to change the display color of gif transparent background? I'm using mpv as player, but it seems that the playing window of Lively doesn't follow my local mpv configurations (I'm not sure)

I'm just flashbanged by a GIF wallpaper which is somehow played in HDR mode. So I'd also like to learn how to disable HDR for just a certain wallpaper or for Lively.

I have quickly looked through the wiki and Issues/Discussions but didn't find anything useful. Sorry in advance if there is something I've missed.

**Comment by rocksdanister** [comment, marked as answer]
You can load your mpv config:
https://github.com/rocksdanister/lively/wiki/Video-Guide#mpv

**Reply by manipEgo** [reply]
Thanks, that really helped.

---

I didn't find this wiki page because I went directly to the [GIF Guide](https://github.com/rocksdanister/lively/wiki/GIF-Guide). It also mentions the video player, but not as detailed, and I didn't expect that there would be another page with more detailed guide about mpv since there is no other link on that page. Maybe the GIF Guide page will need some update?

**Reply by rocksdanister** [reply]
Good point, I update with the same doc to gif guide.
